test/unit/relay/mutation.test.js

```diff
@@ -91,7 +91,9 @@ describe('relay', function () {
             },
             newTaskEdge: {
               type: this.viewerTaskConnection.edgeType,
-              resolve: (payload) => this.viewerTaskConnection.resolveEdge(payload.task)
+              resolve: (payload, args, context, info) => this.viewerTaskConnection.resolveEdge(
+                payload.task, -1, '', {...info, orderAttributes: [this.Task.primaryKeyAttribute]}, payload
```

Comment:
Why is orderAttributes required here? That should be known to the connection

Reply:
Ah nvm, it's ofcourse the requested order by from the query. Maybe it's better if it takes a value directly from the orderBy  setup?

Reply:
@mickhansen problem is we would have to rebuild it from `orderBy` for every single edge, that would be a lot of redundant computation

Reply:
I could use `createSelector` from `reselect` to make a function that returns the `orderAttributes` from `args.orderBy` in a memoized way, without unnecessary recomputation